conda/common/constants.py

```diff
@@ -12,3 +12,5 @@
 NULL = NULL
 
 CONDA_TEMP_EXTENSION = '.c~'
+
+COMSPEC_PATH = "C:\\Windows\\system32\\cmd.exe"
```

Comment:
We can't hardcode it like this.  There's a couple of fallbacks in https://github.com/ContinuumIO/anaconda-distribution-installer/pull/11/files#diff-11d98bb00d3a7be9301a2b03fcd8b9c8R251